src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -2347,6 +2354,13 @@ bool processOnChangeConditionForSubscription
   Restriction                   res;
   ContextElementResponseVector  rawCerV;
 
+  /* Include scopes for entitiesQuery() if q filter is not empty */
+  if (qFilter != "")
+  {
+    Scope* sc = new Scope(SCOPE_TYPE_SIMPLE_QUERY, qFilter);
+    res.scopeVector.push_back(sc);
+  }
```

Comment:
I'm not explicitely releaseing the memory allocated with the `new` in this block, but I understand that as part of the Restriction release process is automatically freed. Or maybe I'm wrong?

(Anyway, check-leak jenkins jobs will warn if I'm wrong)